.clang-tidy

```diff
@@ -46,18 +59,31 @@ CheckOptions:
   - key: readability-identifier-naming.ParameterIgnoredRegexp
     # Allow single-letter uppercase function parameters
     value: "[A-Z]"
+  - key: readability-identifier-naming.ProtectedMemberCase
+    value: lower_case
+  - key: readability-identifier-naming.ProtectedMemberPrefix
+    value: m_
   - key: readability-identifier-naming.PrivateMemberCase
     value: lower_case
   - key: readability-identifier-naming.PrivateMemberPrefix
     value: m_
   - key: readability-identifier-naming.TemplateParameterCase
     value: CamelCase
+  ##########################
   # Other coding conventions
+  ##########################
   - key: readability-braces-around-statements.ShortStatementLines
     # Allow control-flow statements w/o braces when on a single line
     value: 1
+  ##########################
+  # Various check options
+  ##########################
+  - key: misc-const-correctness.TransformPointersAsValues
```

Comment:
From [the documentation](https://clang.llvm.org/extra/clang-tidy/checks/misc/const-correctness.html), the warning on pointers should be enabled with `WarnPointersAsValues` before enabling just the fixit hints with `TransformPointersAsValues`. Not sure if this is technically required since we are obviously receiving warnings already.